packages/fluent/scss/notification/_theme.scss

```diff
@@ -0,0 +1,31 @@
+@use "../core/color-system" as *;
+@use "../core/mixins" as *;
+@use "_variables.scss" as *;
+
+@mixin kendo-notification--theme() {
+
+    // Theme
+    .k-notification {
+        color: var(--INTERNAL--kendo-notification-text, #{$kendo-notification-text});
+        background-color: var(--INTERNAL--kendo-notification-bg, var(--kendo-notification-bg, #{$kendo-notification-bg}));
+        border-color: var(--INTERNAL--kendo-notification-border, var(--kendo-notification-border, #{$kendo-notification-border}));
+    }
+
+    .k-notification-wrap {
+        >.k-i-close {
+            color: inherit;
+        }
+    }
+
+    .k-notification-group .k-notification {
+        box-shadow: var(--INTERNAL--kendo-notification-shadow, var(--kendo-notification-shadow, #{$kendo-notification-shadow}));
```

Comment:
From the design file it seems that there is no shadow for the Fluent Notification. Could you please double check this ?